### PR TITLE
fix(core): handle dynamic height of dynamic page wrapper

### LINKED ISSUE #7567: Bug: Duplicate scrolling bar on Dynamic Page
This issue arises in the Template UI after we built in the MessageStrip. 
image
The issue only occurs the first time you open Java or Fiori template after a page refresh.
It would be great if you could remove the duplicate scrolling bar. 

Please open a PR to [Template UI Repository](https://github.tools.sap/dxp/template-ui) or edit the existing [branch](https://github.tools.sap/dxp/template-ui/pull/200) which can be reviewed by us. 

The page on which it happens is [Template-detail.page.component.ts](https://github.tools.sap/dxp/template-ui/blob/master/src/app/pages/templates-detail-page/templates-detail-page.component.html).

To reproduce it, locally execute the following [branch](https://github.tools.sap/dxp/template-ui/pull/200), open the [Template UI](https://sap.dev.dxp.k8s.ondemand.com/projects/01fgv1yekxw7vfpxtbxdsbmega/components?modal=%2Fprojects%2F01fgv1yekxw7vfpxtbxdsbmega%2Ftemplates%3F~mode%3Dmodal&modalParams=%7B%22title%22:%22Create%20Component%20from%20Template%22%7D) and click on Java or Fiori template.

### PR DESCRIPTION
**Pull request by N1XUS**
## Related Issue(s)

closes #7567 

## Description
Added ability to change size listener from window to the parent element of the dynamic page component.

### Before:
image

### After:
image

##### During Implementation

1. Visual Testing:

-   [x] visual misalignments/updates
-   [x] check Light/Dark/HCB/HCW themes
-   [x] RTL/LTR - proper rendering and labeling
-   [x] responsiveness(resize)
-   [x] Content Density (Cozy/Compact/(Condensed))
-   [x] States - hover/disabled/focused/active/on click/selected/selected hover/press state
-   [x] Interaction/Animation - open/close, expand/collapse, add/remove, check/uncheck
-   [x] Mouse vs. Keyboard support
-   [x] Text Truncation

2. API and functional correctness

-   [x] check for console logs (warnings, errors)
-   [x] API boundary values
-   [x] different combinations of components - free style
-   [x] change the API values during testing

3. Documentation and Example validations

-   [x] missing API documentation or it is not understandable
-   [x] poor examples
-   [x] Stackblitz works for all examples

4. Accessibility testing
5. Browser Testing - Edge, Safari, Chrome, Firefox

##### PR Quality

-   [x] the commit message(s) follows the guideline:
        https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
-   [x] tests for the changes that have been done
-   [x] all items on the PR Review Checklist are addressed :
        https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
-   [x] Run npm run build-pack-library and test in external application
-   [x] update `README.md`
-   [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)

### CODE REVIEW

**Review by dmitry-stepanenko — APPROVED** [review]

File: libs/core/src/lib/dynamic-page/dynamic-page.component.ts [hunk 1]
```diff
@@ -249,12 +252,20 @@ export class DynamicPageComponent implements AfterViewInit, OnDestroy {
...
-            });
+        const listener = this._dynamicPageWrapper ? this._listenToWrapperResize() : this._listenToWindowResize();
+
+        listener.pipe(debounceTime(100), takeUntil(this._onDestroy$)).subscribe(() => {
+            this._setContainerPositions();
+            this._sizeChangeHandle();
+        });
+    }
+
+    private _listenToWrapperResize(): Observable<ResizeObserverEntry[]> {
```
Comment (platon-rov) [inline libs/core/src/lib/dynamic-page/dynamic-page.component.ts:263]: `@hidden`

**Review by platon-rov — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

**Comment by MichaelBaySAP** [comment]
I have updated node_modules to the latest fundamental version. However, the issue is still coming for me 
image
image

**Comment by MichaelBaySAP** [comment]
I have updated node_modules to the latest fundamental version. After applying the following, the issue went away initially. 
image
But it reoccurred, after I pressed "**Run**" and stepped to the Wizard page and back. The wizard page also has the duplicate scroll bar.

**Comment by MichaelBaySAP** [comment]
https://github.com/SAP/fundamental-ngx/issues/7628

**Pull request merged**